keysmap.list

```diff
@@ -1,14 +1,14 @@
 io.swagger.core.v3:swagger-annotations:jar:2.1.7 			        = noSig
 com.fasterxml.jackson.datatype:jackson-datatype-joda:pom:2.12.1     = badSig
 com.fasterxml.jackson.core:jackson-databind:pom:2.12.1 			    = badSig
-com.fasterxml.jackson.core:jackson-annotations:pom:2.12.1 		    = badSig
-com.fasterxml.jackson.core:jackson-core:pom:2.12.1 			        = badSig
-com.fasterxml.jackson.dataformat:jackson-dataformat-xml:pom:2.12.1  = badSig
-com.fasterxml.jackson.datatype:jackson-datatype-joda:pom:2.13.3     = badSig
-com.fasterxml.jackson.core:jackson-databind:pom:2.13.3 			    = badSig
 com.fasterxml.jackson.core:jackson-annotations:pom:2.13.3 		    = badSig
 com.fasterxml.jackson.core:jackson-core:pom:2.13.3 			        = badSig
 com.fasterxml.jackson.dataformat:jackson-dataformat-xml:pom:2.13.3  = badSig
+com.fasterxml.jackson.datatype:jackson-datatype-joda:pom:2.13.3     = badSig
+com.fasterxml.jackson.core:jackson-databind:pom:2.13.3 			    = badSig
+com.fasterxml.jackson.core:jackson-annotations:pom:2.13.4		    = badSig
+com.fasterxml.jackson.core:jackson-core:pom:2.13.4 			        = badSig
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml:pom:2.13.4  = badSig
```

Comment:
Is this supposed to be here? I updated them from 2.13.3 to 2.13.4 in my hotfix update dropwizard PR [here](https://github.com/dockstore/dockstore/pull/5167/files#diff-0e026d685bb906320b5586564b51142e9e5c7a34bfc9c756cde9e427f481cf49) and I saw that you cherry picked this PR to your branch, but I don't see the pom changes. If you didn't mean to cherry pick it, then I think you can just check out the develop version of this file and the build should be okay, unless you need these changes for something else

Reply:
Hmm, I had started this work off the hotfix branch, then realized the ticket wasn't targetted for 1.13x, so changed it to develop, and may have inadvertently brought some stuff over I shouldn't have. I'll take a look.